.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- Good job 👍 adding the node_modules directory but kindly make sure to add the "test.md" as well as per the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20test.md,your%20GitHub%20repository.)

Reply:
@AbelG101  Hi please see my changes and review